src/xrpld/app/tx/detail/Escrow.cpp

```diff
@@ -347,6 +349,27 @@ EscrowFinish::preflight(PreflightContext const& ctx)
         }
     }
 
+    if (ctx.tx.isFieldPresent(sfCredentialIDs))
+    {
+        if (!ctx.rules.enabled(featureCredentials) ||
+            !ctx.rules.enabled(featureDepositPreauth) ||
```

Comment:
You don't need to check for `DepositPreuth` and `DepositAuth`. These amendments are enabled. Same comment applies to `PayChan` and `Payment`.

Reply:
fixed